pkg/executor/import_into.go

```diff
@@ -290,9 +290,11 @@ func (e *ImportIntoExec) importFromSelect(ctx context.Context) error {
 		defer close(selectedRowCh)
 		fields := exec.RetTypes(e.selectExec)
 		var idAllocator int64
+		chkSize := e.selectExec.InitCap()
+		maxChkSize := e.selectExec.MaxChunkSize()
```

Comment:
what's the `MaxChunkSize`?

Reply:
It is the value of `tidb_max_chunk_size`, by default the value is 1024.